doc/source/math/resizing_ops.rst

```diff
@@ -0,0 +1,319 @@
+.. _resizing_ops:
+
+##################
+Resizing Operators
+##################
+
+
+Introduction
+============
+In ODL, resizing of a discretized function is understood as the operation of shrinking or enlarging its domain in such a way that the size of the partition cells do not change.
+This "constant cell size" restriction is intentional since it ensures that the underlying operation can be implemented as array resizing without resampling, thus keeping those two functionalities separate.
+
+
+Basic setting
+=============
+Let now :math:`\mathbb{R}^n` with :math:`n \in \mathbb{N}` be the space of one-dimensional real vectors encoding values of a function defined on an interval :math:`[a, b] \subset \mathbb{R}`.
```

Comment:
Well it's not really long and rather incomplete, not sure it helps much. But it can be linked anyway.